cli/process_utils/process_utils.go

```diff
@@ -165,6 +165,53 @@ func StopProcess(pidFile string) (int, error) {
 	return pid, nil
 }
 
+// QuitProcess quits the process by pidFile.
+func QuitProcess(pidFile string) (int, error) {
+	pid, err := GetPIDFromFile(pidFile)
+	if err != nil {
+		return 0, err
+	}
+
+	alive, err := IsProcessAlive(pid)
+	if !alive {
+		return 0, fmt.Errorf(`the process is already dead. Error: "%v"`, err)
+	}
+
+	if err = syscall.Kill(pid, syscall.SIGQUIT); err != nil {
```

Comment:
As I can see, Stop, Quit, Kill functions have similar code. Can we extract `GetPIDFromFile` and `IsProcessAlive` to separate function. Stop, Quit and Kill must be a function, which accepts PID and performs required operations with the process.